pkg/build/api/validation/validation.go

```diff
@@ -76,6 +76,17 @@ func ValidateBuildConfig(config *buildapi.BuildConfig) fielderrors.ValidationErr
 	}
 
 	allErrs = append(allErrs, validateBuildSpec(&config.Spec.BuildSpec).Prefix("spec")...)
+
+	// validate ImageChangeTriggers of DockerStrategy builds
```

Comment:
@bparees does it make sense here to also try to validate the `ObjectReference`(s), `DockerStrategy.From` and `ImageChange.From`, when they're not `nil`?